community/kernel/src/main/java/org/neo4j/kernel/impl/api/store/StoreNodeRelationshipCursor.java

```diff
@@ -85,7 +86,7 @@ public StoreNodeRelationshipCursor init( boolean isDense,
 
         if ( isDense && relationshipId != Record.NO_NEXT_RELATIONSHIP.intValue() )
         {
-            relationshipGroupStore.getRecord( firstRelId, groupRecord, FORCE );
+            cursors.relationshipGroup().next( firstRelId, groupRecord, FORCE );
```

Comment:
`next()` returns `record.inUse()` but here and in other places we read with `FORCE` mode so we need to read notInUse records and do not really care about `next()` return value